### ISSUE TITLE
Training with scale=1 doesn't do anything

### ISSUE THREAD

**Issue by buildist** [xinntao/EDVR#158]
I'm trying to train a model for video deblocking (output is the same resolution as the input). I set GT_size=LQ_size and HR_in=true, but after training 60k iterations, the output image is always identical to the input. Do any other settings need to be changed?

`#### general settings
name: qt_cinepak_1x
use_tb_logger: false
model: video_base
distortion: sr
scale: 1
gpu_ids: [0]

#### datasets
datasets:
  train:
    name: REDS
    mode: REDS
    interval_list: [1]
    random_reverse: false
    border_mode: false
    dataroot_GT: D:/datasets/vimeo_septuplet/train
    dataroot_LQ: D:/datasets/vimeo_septuplet_cinepak/train
    cache_keys: D:/datasets/vimeo_septuplet/train.pkl

    N_frames: 7
    use_shuffle: true
    n_workers: 8
    batch_size: 8
    GT_size: 256
    LQ_size: 256
    use_flip: True
    use_rot: true
    color: RGB
  val:
    name: Vimeo90K-Test
    mode: video_test
    dataroot_GT: D:/datasets/vimeo_septuplet/test
    dataroot_LQ: D:/datasets/vimeo_septuplet_cinepak/test
    cache_data: True
    N_frames: 7
    padding: new_info

#### network structures
network_G:
  which_model_G: EDVR
  nf: 64
  nframes: 7
  groups: 8
  front_RBs: 5
  back_RBs: 10
  predeblur: false
  HR_in: true
  w_TSA: true

#### path
path:
  # pretrain_model_G: ../experiments/qt_cinepak/models/93000_G.pth
  strict_load: false
  # resume_state: ../experiments/qt_cinepak_1x/training_state/8000.state

#### training settings: learning rate scheme, loss
train:
  lr_G: !!float 4e-4
  lr_scheme: CosineAnnealingLR_Restart
  beta1: 0.9
  beta2: 0.99
  niter: 600000
  ft_tsa_only: 5000
  warmup_iter: -1  # -1: no warm up
  T_period: [50000, 100000, 150000, 150000, 150000]
  restarts: [50000, 150000, 300000, 450000]
  restart_weights: [1, 1, 1, 1]
  eta_min: !!float 1e-7

  pixel_criterion: cb
  pixel_weight: 1.0
  val_freq: !!float 1e3

  manual_seed: 0

#### logger
logger:
  print_freq: 100
  save_checkpoint_freq: !!float 1e3
`
